python/whylogs/api/logger/__init__.py

```diff
@@ -78,6 +78,51 @@ def log_classification_metrics(
     return _log_with_metrics(data=data, metrics=model_performance_metrics, schema=schema, include_data=log_full_data)
 
 
+def log_multi_classification_metrics(
```

Comment:
Should we have a test for this? Also some docstrings with an example (could be the same for both test and docstring)